src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -30,7 +31,9 @@
 namespace Microsoft.VisualStudio.TestPlatform.Client.DesignMode;
 
 /// <summary>
-/// The design mode client.
+/// The design mode client, which listens inside of vstest.console when we are running under an IDE (like VS), and proceses
```

Comment:
With that point of view it is not even correct to say that vstest.console is running, because that is just one possible implementation of this client. But I will change it to make you happy. :)

Reply:
No need to change it unless you fully agree with it :)